include/dsn/cpp/service_app.h

```diff
@@ -69,7 +69,7 @@ namespace dsn
 
         const std::string& name() const { return _name; }
 
-        dsn_gpid gpid() const { return _gpid; }
+        dsn_gpid get_gpid() const { return _gpid; }
```

Comment:
method gpid() is conflict with default constructor of class gpid, and may
cause tricky bugs, for example, if you want call method gpid(), but the
method not exist in the context, then it will call default constructor, and
compiler will not point it out.

On Wed, Jun 15, 2016 at 5:56 PM, Zhenyu Guo <notifications@github.com>
wrote:

> In include/dsn/cpp/service_app.h
> <https://github.com/imzhenyu/rDSN/pull/403#discussion_r67132910>:
>
> > @@ -69,7 +69,7 @@ namespace dsn
> >
> >          const std::string& name() const { return _name; }
> >
> > -        dsn_gpid gpid() const { return _gpid; }
> > +        dsn_gpid get_gpid() const { return _gpid; }
>
> let's be consistent with the above. sometimes we use get because the
> latter is a type name or we want to pair with set, but not in this case.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/imzhenyu/rDSN/pull/403/files/89a41d5e513a99014291d1ad76187ae04459b1ab#r67132910>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgQPWLkLhoKKezwE6IR7JTrsftF61ZPks5qL8w1gaJpZM4I2J8D>
> .
>